services/uploads/src/clamav.js

```diff
@@ -191,10 +191,11 @@ async function uploadAVDefinitions() {
  */
 function scanLocalFile(pathToFile) {
   try {
-    execSync(
+    let avResult = execSync(
       `${constants.PATH_TO_CLAMAV} -v -a --stdout -d /tmp/ ${pathToFile}`
     );
 
+    console.log(avResult.toString()); // eslint-disable-line no-console
```

Comment:
Still need this?

Reply:
Yep this is just a good double check that the virus scan worked and didn't have a wacky error that didn't throw on our end